### ISSUE TITLE
logger.py not finding version?

### ISSUE THREAD

**Issue by s3pp** [oemof/oemof#201]
Hi, 

I am using python version 3.5.2. I pip installed oemof on branch release/v0.1 in a new virtual env without site-packages.
```
 pip install -e git://github.com/oemof/oemof.git@release/v0.1#egg=oemof
```
 In case I'm calling 
```
from oemof.tools import logger
logger.define_logging()
```
I get the following error message
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'loggor'
>>> from oemof.tools import logger
>>> logger.define_logging()
16:37:34-INFO-Path for logging: /home/s3pp/.oemof/log_files
Traceback (most recent call last):
  File "/home/s3pp/.virtualenvs/test/src/oemof/oemof/tools/logger.py", line 65, in define_logging
    check_git_branch()
  File "/home/s3pp/.virtualenvs/test/src/oemof/oemof/tools/logger.py", line 95, in check_git_branch
    f = open(f_commit, "r")
FileNotFoundError: [Errno 2] No such file or directory: '/home/s3pp/.virtualenvs/test/src/oemof/oemof/tools/../../.git/refs/heads/96c3f4e956700a70ba6d73c714e4651ca3b36758'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/s3pp/.virtualenvs/test/src/oemof/oemof/tools/logger.py", line 67, in define_logging
    check_version()
  File "/home/s3pp/.virtualenvs/test/src/oemof/oemof/tools/logger.py", line 72, in check_version
    f = open('../../VERSION', 'r')
FileNotFoundError: [Errno 2] No such file or directory: '../../VERSION'
```

An issue seems to be with line 72 in oemof/oemof/tools/logger.py.
```
 72     f = open('../../VERSION', 'r') 
```
If I provide an absolute path rather than a relative one everything works fine.

Should this be fixed? Does anybody know what is going wrong?

Best regards,

s3pp

**Comment by uvchik** [comment]
I think I fixed the path with commit 7b1858040. Does it work now?

**Comment by s3pp** [comment]
Thank you very much! Fix is working! Everything works fine now! : )
